fir_notifications/templates/fir_notifications/preferences.html

```diff
@@ -0,0 +1,39 @@
+{% load i18n %}
```

Comment:
The interface to manage notification subscriptions is not great at the moment.

I think it would be better to have a "Add subscription" button, with a form to chose Incident/Event, the standard tag input used for business lines for BLs and events, a field for the method, and potentially additional fields for the template and method options.

This could be done later though.